### PR TITLE
Update README with notes for using f90nml

### PR DESCRIPTION
**Pull request by MaggieMarvin**
Added notes and an example for using the f90nml CLI to insert multiple input filenames into the namelist at once

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +125,10 @@ Namelist Option : `file_vars`  Full name of input file (Supports either text or
 - See example file inputs for variables and format (`gfs.t12z.20220701.sfcf000.canopy.txt` or `gfs.t12z.20220701.sfcf000.canopy.nc`).  Example surface met/land/soil inputs are based on NOAA's UFS-GFSv16 inputs initialized on July 01, 2022 @ 12 UTC (forecast at hour 000). Other external inputs for canopy related and other calculated variables are from numerous sources.  See [Table 2](#table-2-canopy-app-required-input-variables) below for more information.  **Note:** The example GFSv16 domain has been cut to the southeast U.S. region only in this example for size/time constraints here.
 - Canopy-App assumes the NetCDF input files are in CF-Convention and test file is based on UFS-GFSv16; recommend using double or float for real variables.  Input data must be valid values.
 - Canopy-App can also be run with a single point of 1D input data in a text file (e.g. `point_file_20220701.sfcf000.txt`).
+- The namelist can be modified with f90nml (included with the Canopy-App Conda environment) to insert multiple input filenames at once:
```
Comment (drnimbusrain) [inline README.md]:
```suggestion
- The namelist can be modified with f90nml (included with the [Canopy-App Conda](https://github.com/noaa-oar-arl/canopy-app/blob/develop/python/environment.yml) environment) to insert multiple input filenames at once:
```

Comment (drnimbusrain) [reply README.md]: @MaggieMarvin  I added the inserted hyperlink to point to the environment file for further clarification.

**Review by drnimbusrain — CHANGES_REQUESTED** [review]
@MaggieMarvin Looks great, I added one small comment.

File: README.md (hunk 1 above)
Comment (zmoon) [reply README.md]: Relative path (`./python/environment.yml`) is another option for the link target.

File: README.md [hunk 2]
```diff
@@ -125,6 +125,10 @@ Namelist Option : `file_vars`  Full name of input file (Supports either text or
...
+```
+f90nml -g filenames -v file_vars="$(realpath *.txt | xargs -I {} echo "'{}'")" namelist.canopy namelist.canopy_copy
+```
```
Comment (zmoon) [inline README.md]: indent so it is included in the block

````suggestion
  ```
  f90nml -g filenames -v file_vars="$(realpath *.txt | xargs -I {} echo "'{}'")" namelist.canopy namelist.canopy_copy
  ```
````

File: README.md (hunk 1 above)
Comment (zmoon) [reply README.md]: Also could link https://f90nml.readthedocs.io/en/latest/cli.html for f90nml as well?

**Comment by MaggieMarvin** [comment]
Thanks both! I've incorporated your changes, also linking f90nml to its readsthedocs page

**Review by drnimbusrain — APPROVED** [review]

**Pull request merged**